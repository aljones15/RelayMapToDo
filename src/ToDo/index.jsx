import React from 'react';
import style from './style';
import { graphql, createFragmentContainer } from 'react-relay';
import { css } from 'aphrodite';

const ToDo = createFragmentContainer(
  ({ data }) => {
    return (
      <li className={css(style.ToDoContainer)}>
        {data._id} | {data.text} | {data.likes}
      </li>
    );
  },
  graphql`
    fragment ToDo on ToDo {
      text
      likes
      _id
    }
  `
);

export default ToDo;
