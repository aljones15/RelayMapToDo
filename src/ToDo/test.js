import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import ToDo from './index';

const todo = {
  text: 'test',
  likes: 1,
  _id: 1,
};

describe('ToDo Should', () => {
  const elJson = ReactTestRenderer.create(<ToDo data={todo} />).toJSON();
  it('match a snapshot', () => {
    expect(elJson).toMatchSnapshot();
  });
});
