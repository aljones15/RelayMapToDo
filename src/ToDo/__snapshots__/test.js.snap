// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToDo Should match a snapshot 1`] = `
<li
  className="ToDo_gfjdxk"
>
  1
   | 
  test
   | 
  1
</li>
`;
